ui/app/models/mfa-login-enforcement.js

```diff
@@ -109,7 +109,7 @@ export default class MfaLoginEnforcementModel extends Model {
   }
 
   iconForMount(type) {
-    const mountableMethods = methods();
+    const mountableMethods = filterEnginesByMountType('auth');
```

Comment:
Gut check me, I have includeEnterprise as false, but it kind of feels like it should be true? 
```suggestion
    const mountableMethods = filterEnginesByMountType({ mountType = 'auth', includeEnterprise = false });
```

Reply:
MFA login enforcement is an enterprise feature that supports the following auth methods: userpass, ldap, okta and jwt/oidc. These supported methods are not enterprise only which is why I think this works, but if we use false for includeEnterprise, then I think we should leave a comment.